README.md

```diff
@@ -649,6 +649,16 @@ All statements above will return false if used with `===`
 
 🔗 [**Read More: Use an LTS release of Node.js**](/sections/production/LTSrelease.md)
 
+<br/><br/>
+
+## ![✔] 5.18. Log Routing
```

Comment:
Most of our bullets start with verb/action.

Like 'Don't route logs within the app', or 'Don't write to log files, prefer stdout' (the last sounds very intrigue as many people would be surprised)